### DISCUSSION TITLE
Is it possible to run GDPS in a RPi 4B?

### DISCUSSION THREAD

**Discussion by libracrumbs** [Cvolton/GMDprivateServer#786] [Q&A] [answered]
It would be nice if it worked on Ubuntu MATE or Raspbian. If it does, is it possible to fully set it up?

**Comment by Cvolton** [comment, marked as answer]
it's fully possible, I use a Pi 3B+ to run mine, the setup is the same as on any other Linux machine

**Comment by ghost** [comment]
Raspberry Pi's work like any other web server, and database
